Modules/Core/Common/include/itkSize.h

```diff
@@ -471,6 +471,16 @@ swap(Size<VDimension> & one, Size<VDimension> & two)
   std::swap(one.m_InternalArray, two.m_InternalArray);
 }
 
+
+/** Makes a Size object, having the specified size values. */
+template <typename... T>
+constexpr auto
+MakeSize(const T... values)
+{
+  return Size<sizeof...(T)>{ { static_cast<SizeValueType>(values)... } };
```

Comment:
@blowekamp You wrote:
> These implementations implicitly does explicit type conversion for the arguments, which might hide some unintended conversion issues?

The `static_cast<SizeValueType>` here in `MakeSize` allows users to pass `int` values, as in `MakeSize(1, 2, 3)`.

We could remove this `static_cast<SizeValueType>`, but then the user would have to write something like `MakeSize(1U, 2U, 3U)`, to avoid signed-to-unsigned conversion. What do you think?